KitePrintSDK/src/main/java/ly/kite/checkout/PaymentActivity.java

```diff
@@ -161,15 +202,25 @@ public boolean onMenuItemSelected(int featureId, MenuItem item) {
     }
 
     public void onButtonPayWithPayPalClicked(View view) {
-        PayPalPayment payment = new PayPalPayment(printOrder.getCost(printOrder.getCurrencyCode()), printOrder.getCurrencyCode(), "Product");
-        Intent intent = new Intent(this, com.paypal.android.sdk.payments.PaymentActivity.class);
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYPAL_ENVIRONMENT, printEnvironment.getPayPalEnvironment());
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_CLIENT_ID, printEnvironment.getPayPalClientId());
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYER_ID, "<someuser@somedomain.com>");
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_RECEIVER_EMAIL, printEnvironment.getPayPalReceiverEmail());
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYMENT, payment);
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_SKIP_CREDIT_CARD, true);
-        startActivityForResult(intent, REQUEST_CODE_PAYPAL);
+
+//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYPAL_ENVIRONMENT, printEnvironment.getPayPalEnvironment());
+//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_CLIENT_ID, printEnvironment.getPayPalClientId());
+//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYER_ID, "<someuser@somedomain.com>");
```

Comment:
Ah `PayPalConfiguration` has an `acceptCreditCards` method  -- http://stackoverflow.com/questions/18515960/disable-credit-card-payment-option-in-paypal-integration